k8s/demo/common/ccd/latest-ccd-chart.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: ccd-demo
   namespace: ccd
   annotations:
-    flux.weave.works/automated: 'false'
+    flux.weave.works/automated: 'true'
```

Comment:
having this to true was giving problems to:
      image: hmctspublic.azurecr.io/ccd/case-management-web:pr-827
Please leave it as false for now

Reply:
ok reverted